src/transport/SessionManager.h

```diff
@@ -237,9 +237,8 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate
     // is returned. Otherwise, an Optional<SessionHandle> with no value set is returned.
     //
     //
-    Optional<SessionHandle>
-    FindSecureSessionForNode(ScopedNodeId peerNodeId,
-                             Transport::SecureSession::Type type = Transport::SecureSession::Type::kUndefined);
+    Optional<SessionHandle> FindSecureSessionForNode(ScopedNodeId peerNodeId,
```

Comment:
The description of this function should be updated to indicate that `kUndefined` is to no longer be passed in.

Reply:
Good catch, done.